Dockerfile

```diff
@@ -53,6 +54,11 @@ RUN curl -O -L https://github.com/INCATools/dosdp-tools/releases/download/v$DOSD
     && chmod +x /tools/dosdp-tools-$DOSDP
 ENV PATH "/tools/dosdp-tools-$DOSDP/bin:$PATH"
 
+###### RELATION-GRAPH ######
+RUN curl -O -L https://github.com/balhoff/relation-graph/releases/download/v$RELGRAPH/relation-graph-$RELGRAPH.tgz \
+    && tar -zxf relation-graph-$RELGRAPH.tgz
```

Comment:
Will we need to provide required permissions with `chmod` ?

Reply:
I think it must come set correctly out of the zip, since the `chmod` commands on the other lines are not referencing the executable anyway: `chmod +x /tools/dosdp-tools-$DOSDP` (this is the folder).